UI/SciMaterials.UI.MVC/appsettings.json

```
@@ -11,7 +11,7 @@
     "SQLiteConnectionString": "Data Source=.\\SciMaterials.db"
   },
   "DbSettings": {
-    "DbProvider": "SQLite",
+    "DbProvider": "SqlServer",

```

```suggestion
    "DbProvider": "SQLite",

```
Возвращаем обратно для обеспечения расширеной поддержки БД на разных рабочих местах.